src/TestInfrastructure/Framework/ConfigurableIntegrationSettings.cs

```diff
@@ -25,7 +25,5 @@ public class ConfigurableSonarLintSettings : ISonarLintSettings
         public DaemonLogLevel DaemonLogLevel { get; set; }
 
         public bool IsActivateMoreEnabled { get; set; }
-
-        public bool SkipActivateMoreDialog { get; set; }
```

Comment:
Miscellaneous change: the `SkipActiveMoreDialog` was removed from the public interface a long time ago, but was left behind in this test class.